### ISSUE TITLE
Google Authenticator input field looks ackward

### ISSUE THREAD

**Issue by ArthurBorsboom** [osTicket/osTicket-plugins#192]
Although the Google Authenticator plugin works as expected, the login form input field for the agents looks a weird on Chromium.

The out of the box login form looks good.
![2020-12-30-104354_1044x596_scrot](https://user-images.githubusercontent.com/6639859/103343330-231e5780-4a8c-11eb-93fa-18250dc3069a.png)

The Google Authentictor validation after the agent login field looks too long.
![2020-12-30-104430_875x599_scrot](https://user-images.githubusercontent.com/6639859/103343402-447f4380-4a8c-11eb-954c-a3a0d709b2b6.png)

**Comment by JediKev** [comment]
@ArthurBorsboom 

Yes, the textbox is within a `<form>` tag so it's aligned within that element. We can look at adding a `transform` property or something to scoot it over.

Cheers.

**Comment by JediKev** [comment]
@ArthurBorsboom 

To test please inspect the page in Chromium, find and click on the `<input>` of the textbox, go to the Styles tab, and put the following property `transform: translateX(-8%);`. Once you do this it should be moved over to the left.

Cheers.

**Comment by ArthurBorsboom** [comment]
It looks definitely better, but it is not perfectly centered (if that was the aim).

![2020-12-30-163523_1687x586_scrot](https://user-images.githubusercontent.com/6639859/103364096-2aaa2480-4abd-11eb-8512-59f31368a05e.png)

**Comment by JediKev** [comment]
@ArthurBorsboom 

Okay, your box is a little different than mine for some reason. We will have to look further into this to come up with a better solution.

Cheers.

**Comment by ArthurBorsboom** [comment]
There are two 'customizations' in the login form.

One is the company logo and the other one is the background image.
Both are uploaded by the normal web interface of osTicket.

Maybe this influences the layout.
